<?xml version="1.0" encoding="UTF-8"?>
<!--
SPDX-FileCopyrightText: 2019-2023 Mattia Basaglia <dev@dragon.best>

SPDX-License-Identifier: GPL-3.0-or-later
-->
<ui version="4.0">
 <class>dock_script_console</class>
 <widget class="QDockWidget" name="dock_script_console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="utilities-terminal"/>
  </property>
  <property name="windowTitle">
   <string>Script Console</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="console_output">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="KHistoryComboBox" name="console_input">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_clear_console">
        <property name="toolTip">
         <string>Clear Output</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="icon">
         <iconset theme="edit-clear-all"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="check_persist">
        <property name="toolTip">
         <string>Persist Logs</string>
        </property>
        <property name="text">
         <string>Persist</string>
        </property>
        <property name="icon">
         <iconset theme="pin"/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_reload">
        <property name="toolTip">
         <string>Reload Script Modules</string>
        </property>
        <property name="text">
         <string>Reload</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="console_language"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
